smart_card_connector_app/src/background.js

```diff
@@ -165,6 +166,28 @@ if (GSC.ExecutableModule.TOOLCHAIN ===
   GSC.BackgroundPageUnloadPreventing.enable();
 }
 
+if (chrome.smartCardProviderPrivate !== undefined) {
+  const HANDLE_CHROME_REQUESTS = 'handle_requests_from_chrome_smart_card_api';
+  chrome.storage.managed.get(HANDLE_CHROME_REQUESTS, (policies) => {
```

Comment:
The policy can also change at runtime (it's especially possible during the first-time login). So it's better to set up an observer that'd react to the changes of the policy value.

Reply:
fixed